frontend/src/pages/mentoringUpdate/components/MentoringUpdateForm/MentoringUpdateForm.tsx

```diff
@@ -0,0 +1,227 @@
+import { putMentoring } from '../../apis/putMentoring';
+import { isInitialMentoringData } from '../../utils/isInitialMentoringData';
+
+import type { MentoringUpdateFormData } from '../../types/mentoringUpdateForm';
+
+function MentoringUpdateForm() {
+  const [mentoringData, setMentoringData] = useState<MentoringUpdateFormData>({
+    price: 0,
+    category: [],
+    introduction: '',
+    career: 0,
+    content: '',
+    profileImageUrl: '',
+    certificateInfos: [
+      {
+        id: '0',
+        title: null,
+        type: null,
+      },
+    ],
+  });
+  const [profileImageFile, setProfileImageFile] = useState<File | null>(null);
+  const [certificateImageFiles, setCertificateImageFiles] = useState<File[]>(
+    [],
+  );
+  const initialCertificatesIdRef = useRef<string[]>([]);
+
+  const priceErrorMessage = priceValidator(mentoringData.price);
+  const introduceErrorMessage = introduceValidator(mentoringData.introduction);
+  const careerErrorMessage = careerValidator(mentoringData.career);
+
+  const handleMentoringDataChange = (
+    newData: Partial<MentoringUpdateFormData>,
+  ) => {
+    setMentoringData((prevData) => ({
+      ...prevData,
+      ...newData,
+    }));
+  };
+
+  const handleProfileImageChange = (file: File | null) => {
+    setProfileImageFile(file);
+    setMentoringData((prev) => ({ ...prev, profileImageUrl: null }));
+  };
+
+  const handleCertificateImageFilesChange = (files: File[]) => {
+    setCertificateImageFiles(files);
+  };
+
+  const { mentoringId } = useParams();
+
+  const submitMentoringForm = async () => {
+    const unchangedCertificates = mentoringData.certificateInfos.filter(
+      (e) => !initialCertificatesIdRef.current.includes(e.id),
+    );
+
+    try {
+      const response = await putMentoring({
+        mentoringData: {
+          ...mentoringData,
+          certificateInfos: unchangedCertificates.map(
+            (unchangedCertificate) => ({
+              title: unchangedCertificate.title,
+              type: unchangedCertificate.type,
+            }),
+          ),
+          profileImageUrl: mentoringData.profileImageUrl,
+        },
+        profileImageFile,
+        certificateImageFiles,
+        mentoringId: mentoringId!,
+      });
+      if (response.status === 200) {
+        alert('멘토링 수정 성공');
+      }
+    } catch (error) {
+      console.error('멘토링 수정 실패');
+      Sentry.captureException(error, {
+        level: 'warning',
+        tags: {
+          feature: 'mentoring',
+          step: 'mentoring-update',
+        },
+      });
+    }
+  };
+
+  const navigate = useNavigate();
+
+  const handleSubmitButtonClick = async (
+    e: React.FormEvent<HTMLFormElement>,
+  ) => {
+    e.preventDefault();
+    if (priceErrorMessage || introduceErrorMessage || careerErrorMessage) {
+      alert('입력값을 확인해주세요.');
+      return;
+    }
+    await submitMentoringForm();
+    navigate(PAGE_URL.HOME, { state: { refetch: true } });
```

Comment:
p5: refetch 부분은 없어져도 될 거 같아 👍

```suggestion
    navigate(PAGE_URL.HOME);
```

Reply:
navigate를 호출하는 위치를 변경하면서 제거했어!

Reply:
[refactor: 성공시 페이지 이동하게 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/496/commits/cd1f7b2cddabcaaca288a7aa249c78f7d0941780)